File: StykerDG.FarmForge.DataModel/Models/LogType.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace StrykerDG.FarmForge.DataModel.Models
{
    [Table("LogType")]
    public class LogType : BaseModel 
    {
        [Key]
        public int LogTypeId { get; set; }
        public string Name { get; set; }
        public string Label { get; set; }
        public string Description { get; set; }
    }
}

File: StrykerDG.FarmForge.Client/farmforge_client/lib/models/general/log_type.dart
class LogType {
  int logTypeId;
  String entityType;
  String name; 
  String label;
  String description;

  LogType({
    this.logTypeId,
    this.entityType,
    this.name,
    this.label,
    this.description
  });

  factory LogType.fromMap(Map<String, dynamic> data) {
    return LogType(
      logTypeId: data['LogTypeId'],
      entityType: data['EntityType'],
      name: data['Name'],
      label: data['Label'],
      description: data['Description']
    );
  }

  Map<String, dynamic> toMap() {
    return {
      'LogTypeId': this.logTypeId,
      'EntityType': this.entityType,
      'Name': this.name,
      'Label': this.label,
      'Description': this.description
    };
  }
}